src/index.css

```diff
@@ -84,27 +84,29 @@ code {
   position: absolute;
   z-index: 1;
   bottom: 42px;
-  left: calc(50% - (3 / 2) * (32px + 2 * 15px));
+  left: calc(50%);
   background-color: #fafafa;
   border-radius: 6.8px;
   box-shadow: 0px 1.7px 10.2px rgba(139, 139, 139, 0.36);
   display: flex;
   flex-direction: row;
+  width: 111px;
+  height: 38px;
 }
 
 .camera-button {
   background: none;
   border: none;
 
-  padding: 10px 15px;
+  padding: 5px 8.5px;
   display: flex;
   flex-direction: column;
   align-items: center;
 
-  font-size: 12px;
   font-weight: normal;
-  line-height: 14px;
-  color: #333;
+  font-size: 7px;
+  line-height: 8px;
+  color: #333333;
```

Comment:
There was no reason to change this from `#333` to `#333333`, these are equivalent in CSS